### DISCUSSION TITLE
XAML does not display accented letters or the letter ñ in Windows

### DISCUSSION THREAD

**Discussion by tenderodigital** [unoplatform/uno#13504] [Q&A] [answered]
When I create a new Uno Platform application I cannot display accented letters or the letter ñ with texts in Spanish.  
It only happens in the Windows version. With Android, Linux or Web it does not happen, in these versions if the texts are displayed well in XAML. 
For example, I create an application with the default template. On MainPage.xml I write:

<Page x:Class="Ejemplo1.MainPage"
	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
	xmlns:local="using:Ejemplo1"
	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
	mc:Ignorable="d"
	Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">

	<StackPanel HorizontalAlignment="Center"
				VerticalAlignment="Center">
		<TextBlock AutomationProperties.AutomationId="HelloTextBlock"
				   Text="Hello Uno Platform José Núñez Martínez" />
	</StackPanel>
</Page>

If I run the Windows version it looks like this:

![image](https://github.com/unoplatform/uno/assets/135991332/85d3a085-4439-42b8-98b3-ae40d15d8b19)

If I create a Winui application in C# (not from Uno Platform) it works fine.  

In case it is of interest when creating the Uno Platform application from the templates, the English (en) and Arabic (ar) language always appear in the configuration. Even forcing to use Spanish as the default language the error continues.

If I use translated strings in a resource file, accents and the letter ñ appear. The problem I have with fields that the user fills, such as the name of the connected user, there I can not use translations in resource file. 

Can you think of a solution or a test I can do?

**Comment by limeniye** [comment]
Indeed. I`ve tested it. It works from code:
```C#
SymbolTextBlock.Text = "ñ";
```
But not works from XAML.

**Reply by jeromelaban** [reply, marked as answer]
@tenderodigital thanks for the report. This is indeed interesting and would mean that the default encoding of the files is incorrect. You can try changing the encoding of XAML files by using "save" as them as unicode or UTF-8 with header:
![image](https://github.com/unoplatform/uno/assets/5839577/c1178a8a-2e12-4390-8fbe-3a9050473fbe)

Let's keep track in https://github.com/unoplatform/uno.templates/issues/263

**Comment by tenderodigital** [comment]
Hello: 
thanks for the answer, I tried to save as UTF-8 with signature and now if it works well.
